CHANGES.md

```diff
@@ -1,3 +1,16 @@
+## DC/OS 1.11.5
```

Comment:
Thanks, @timaa2k 

@orsenthil you probably agree that we should sort this out in a standardized "version bump" step, where we do not only bump the version in gen/dcos-config.yaml etc, but also do all other things (like adding new sections here in CHANGES.md) that need to be done after cutting a release.

Reply:
Yup, @jgehrcke - This needs to sorted out. Perhaps as soon as the minor version branch is created. Add a section to the existing `CHANGES.md`